framework_tvb/tvb/core/services/encryption_handler.py

```diff
@@ -40,6 +40,7 @@
 import pyAesCrypt
 from tvb.basic.logger.builder import get_logger
 from tvb.basic.profile import TvbProfile
+from tvb.interfaces.command.demos.importers.encrypt_data import generate_random_password
```

Comment:
This dependency is not correct (`interfaces` package can depend on `core`, nor reverse). 
Maybe put function `generate_random_password` here and use it in the interfaces ?

Reply:
Okay, but I had to make it a static method in EncryptionHandler and send the pass_size as parameter instead of using self.pass_size in order to avoid initializing an EncryptionHandler object, which would result in an error.